switch:
    ## Data Process
    kLidarFlatProcess: false
    kFisheyeFlatProcess: false

    ## Registration, Colorization and Mapping
    # view
    kCreateDensePcd: true
    kViewRegistration: false
    kFullViewMapping: true

    ## Calibration and Optimization cost analysis
    kCeresOptimization: false
    kMultiSpotsOptimization: false
    kParamsAnalysis: false
    
    # spot
    kSpotRegistration: false
    kGlobalMapping: false

    # colorization
    kFullViewColorization: false
    kGlobalColoredMapping: false

spot:
    kOneSpot: -1  # -1: means run all the spots, other means run the specific spot index 

essential:
    kLidarTopic: "/livox/lidar"
    kDatasetName: "lh3_global"
    kNumSpots: 5
    # kDatasetName: "crf"
    # kNumSpots: 1
    # kDatasetName: "rb1"
    # kNumSpots: 1
    # kDatasetName: "parking"
    # kNumSpots: 6
    kNumViews: 5
    kFisheyeRows: 2048
    kFisheyeCols: 2448
    kAngleInit: -50
    kAngleStep: 25

transform:
    rx: 0.012864
    ry: -0.001143
    rz: 0.160874
    tx: 0.299831
    ty: 0.230617
    tz: -0.029438
